src/common/libflux/stats.h

```diff
@@ -0,0 +1,55 @@
+/************************************************************\
+ * Copyright 2014 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+#ifndef _FLUX_CORE_STATS_H
+#define _FLUX_CORE_STATS_H
+
+#include <stdarg.h>
+
+#include "handle.h"
+
+/* Format a metric name and increment its value by '1'.
+ */
+int flux_stats_inc (flux_t *h, const char *fmt, ...)
```

Comment:
Just wondering, should all these functions return `void`?  It seems unlikely that callers will want to be burdened with handling any potential error.